.prettierrc.js

```diff
@@ -1,12 +1,11 @@
 /** @type {import("prettier").Config} */
+// TODO: Re-enable tailwind prettier plugin
 module.exports = {
   printWidth: 80,
   tabWidth: 2,
   semi: false,
   singleQuote: true,
   quoteProps: 'consistent',
   jsxSingleQuote: false,
-  trailingComma: 'none',
-  plugins: ['prettier-plugin-tailwindcss'],
-  tailwindConfig: './packages/wallet/frontend/tailwind.config.js'
+  trailingComma: 'none'
```

Comment:
Disabling it for now. Running Prettier will format all the files where we have the previous Tailwind classes (like `text-green-3`) since they are not present in the config.